### DISCUSSION TITLE
Web Clipper needs update to adapt to Chrome manifest V3

### DISCUSSION THREAD

**Discussion by Nriver** [TriliumNext/Notes#292] [Development]
In Chrome 127, there is a warning says that Trilium Web Clipper will not be supported soon. I suppose this has something to do with the `Manifest V3` thing.

**Comment by meichthys** [comment]
I think @perfectra1n Is on the case!
https://matrix.to/#/!cQQvgtCfVJtiPWCEXO:matrix.org/$f_W5R6xSVEbA_MFUrAwWJjxXTdFoGGi2scTh4whrJQs?via=matrix.org&via=perfectra1n.com&via=kde.org

**Comment by meichthys** [comment]
Also related: https://github.com/TriliumNext/Notes/issues/128
Currently we don't have issues enabled for the `web-clipper` repository, so i'll convert this to an issue in the Notes repository for now and add a `web-clipper` label.

**Comment by perfectra1n** [comment]
Yeah, I'm working on it. I didn't know that V2 was so old at this point, but they've changed a lot in V3. Hopefully I can easily port it, if not, I'll need to pull in someone else to help me with the JS, but I should be able to at least get it to compile / update to V3.

**Comment by meichthys** [comment]
Closing discussion here. let's track this in #293
